tiledb/sm/array_schema/array_schema.h

```diff
@@ -444,6 +444,12 @@ class ArraySchema {
   /** Generates a new array schema URI with specified timestamp range. */
   Status generate_uri(const std::pair<uint64_t, uint64_t>& timestamp_range);
 
+  /**
+   * Returns the name of the attribute in this schema with a bitsort filter.
+   * If none exists, then this function returns std::nullopt.
```

Comment:
Let's add a note here mentioning the reason we can only have one bitsort attribute. It's obvious to us now but might not be to a new reader.